gui-tk/wiring.tcl

```diff
@@ -499,7 +499,8 @@ proc canvasContext {x y X Y} {
     .wiring.context add command -label "Cut" -command cut
     .wiring.context add command -label "Copy" -command minsky.copy
     .wiring.context add command -label "Save selection as" -command saveSelection
-    .wiring.context add command -label "Paste" -command {paste}
+    .wiring.context add command -label "Paste Group" -command minsky.pasteGroup
+    .wiring.context add command -label "Paste Item(s)" -command minsky.pasteItems
```

Comment:
I would think we just want pastItems, and remove pasteGroup altogether. What do you think?

Reply:
How about making "PasteItems" just plain "Paste" and if I can't get one function to paste both groups and separate items correctly, I can maybe add a "Paste Special" with a popup menu for pasting groups which were groups before they were copied?

Reply:
We are back to using just one paste() function.